src/libraries/Microsoft.PowerFx.Core/Utils/DName.cs

```diff
@@ -143,6 +143,8 @@ public static bool IsValidDName(string strName)
             return false;
         }
 
+        // $$$ Needs optimization $$$
```

Comment:
>// $$$ Needs optimization $$$ [](http://example.com/codeflow?start=8&length=29)

is this showing up on a profiler?

Reply:
Oh yes!

Reply:
Can you file an issue to track it?  
And then include that issue in the comments (and we don't need a $$$ for it then)

Reply:
Created issue #1025